### PR TITLE
Tickets updation implemented

### PR DESCRIPTION
**Pull request by vijay1172**
WhatsApp Image 2024-07-06 at 09 59 44_399c5813
WhatsApp Image 2024-07-06 at 10 00 03_c2c6ba80

### CODE REVIEW

**Comment by Jha2508** [comment]
PR looks good, please resolve the conflict from dev.

**Comment by Jha2508** [comment]
Also there are some UI issues, in the screen shot
resolve them all before getting the approval

**Comment by vijay1172** [comment]
All conflicts resolved and fixed the css issue,  approval of merge requested  @Jha2508

**Review by Jha2508 — APPROVED** [review]

**Pull request merged**
